allensdk/brain_observatory/behavior/data_objects/metadata/behavior_ophys_metadata.py

```diff
@@ -60,6 +60,13 @@ def from_lims(cls, ophys_experiment_id: int,
             ophys_metadata = OphysExperimentMetadata.from_lims(
                 ophys_experiment_id=ophys_experiment_id, lims_db=lims_db)
 
+        if ophys_metadata.project_code != behavior_metadata.project_code:
+            raise ValueError(
```

Comment:
This is inconsistent with the project table, in which you just issue a warning. I think a warning should be ok.

Reply:
Switched to warning.